test/framework/management_cluster.go

```diff
@@ -158,3 +166,87 @@ func WaitForDeploymentsAvailable(ctx context.Context, input WaitForDeploymentsAv
 
 	}, intervals...).Should(BeTrue(), "Deployment %s/%s failed to get status.Available = True condition", input.Deployment.GetNamespace(), input.Deployment.GetName())
 }
+
+// CreateNamespaceInput is the input type for CreateNamespace.
+type CreateNamespaceInput struct {
+	Creator Creator
+	Name    string
+}
+
+// CreateNamespace is used to create a namespace object.
+func CreateNamespace(ctx context.Context, input CreateNamespaceInput, intervals ...interface{}) *corev1.Namespace {
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: input.Name,
+		},
+	}
+	By(fmt.Sprintf("Creating namespace %s", input.Name))
```

Comment:
To help troubleshoot issues if a consumer fails to pass in a `Creator`:
```suggestion
	Expect(input.Creator).NotTo(BeNil(), "input.Creator is required for CreateNamespace")
	By(fmt.Sprintf("Creating namespace %s", input.Name))
```

Reply:
done!